packages/edit-site/src/plugins/index.js

```diff
@@ -17,6 +19,15 @@ registerPlugin( 'edit-site', {
 				<ToolsMoreMenuGroup>
 					<SiteExport />
 					<WelcomeGuideMenuItem />
+					<MenuItem
+						href={ __(
+							'https://wordpress.org/support/article/site-editor/'
```

Comment:
This page doesn't exist right now.

Reply:
Currently, it's a draft but should be published before 5.9 - https://github.com/WordPress/Documentation-Issue-Tracker/issues/93.

Reply:
Hm okay, I think we had discussed this before again in another PR 😄